pyEpiabm/pyEpiabm/tests/test_unit/test_sweep/test_spatial_sweep.py

```diff
@@ -14,69 +15,134 @@ class TestSpatialSweep(TestMockedLogs):
     """
 
     def setUp(self):
+        # 1st Population
+        # 3 cell population
+        # 1st cell: 1 microcell, 100 people, includes the infector
+        # who has InfectionStatus of mildly infected
+        # 2nd cell: 1 microcell, 1 person, includes the infectee
+        # who is Susceptible
+        # 3rd cell: 1 microcell, 1 person who has the status recovered
         self.pop = Population()
-        self.pop.add_cells(1)
+        self.pop.add_cells(2)
         self.cell_inf = self.pop.cells[0]
+        self.cell_susc = self.pop.cells[1]
+        # self.cell_non_susc = self.pop.cells[2]
 
         self.cell_inf.add_microcells(1)
         self.microcell_inf = self.cell_inf.microcells[0]
 
+        self.cell_susc.add_microcells(1)
+        self.microcell_susc = self.cell_susc.microcells[0]
+
+        # self.cell_non_susc.add_microcells(1)
+        # self.microcell_non_susc = self.cell_non_susc.microcells[0]
+
         self.microcell_inf.add_people(100)
         self.infector = self.microcell_inf.persons[0]
+        self.infector.update_status(InfectionStatus.InfectMild)
+
+        self.microcell_susc.add_people(1)
+        self.infectee = self.microcell_susc.persons[0]
+
```

Comment:
Is the default infectious status Susceptible? It's better to specify to make it more clearly.